action.yml

```
@@ -27,6 +27,13 @@ inputs:
     description: 'Python Version specifier (PEP440) - e.g. "21.5b1"'
     required: false
     default: ""
+outputs:
+  is_formatted:
+    description: "Whether the files were formatted using the black formatter."
```

```suggestion
    description: "Files checked are formatted to the current black formatter style."
```
Just trying to be clear - Happy for variations of suggestions.

---

Ill amend the patch, changed made in new version..